dataservice/api/__init__.py

```diff
@@ -1,37 +1,41 @@
 from flask import Blueprint
-from flask_restplus import Api
-from dataservice.api.participant import participant_api
-from dataservice.api.status import status_api
-from dataservice.utils import _get_version
+from dataservice.api.status import StatusAPI
+from dataservice.api.participant import ParticipantAPI
 
-api_v1 = Blueprint('api', __name__, url_prefix='')
 
-api = Api(api_v1,
-          title='Kids First Data Service',
-          description=open('dataservice/api/README.md').read(),
-          version=_get_version(),
-          default='',
-          default_label='')
+def register_crud_resource(app, view, endpoint, url,
```

Comment:
Nice, I like it. I agree we could probably move this chunk of code (method and registering of routes) somewhere later, maybe a view factory or something. But for now it's probably fine.